OAuth/ResourceOwner/DropboxResourceOwner.php

```diff
@@ -38,9 +41,47 @@ protected function configureOptions(OptionsResolver $resolver)
         parent::configureOptions($resolver);
 
         $resolver->setDefaults(array(
-            'authorization_url' => 'https://www.dropbox.com/1/oauth2/authorize',
-            'access_token_url' => 'https://api.dropbox.com/1/oauth2/token',
-            'infos_url' => 'https://api.dropbox.com/1/account/info',
+            'authorization_url' => 'https://www.dropbox.com/oauth2/authorize',
+            'access_token_url' => 'https://api.dropbox.com/oauth2/token',
+            'infos_url' => 'https://api.dropboxapi.com/2/users/get_current_account',
         ));
     }
+
+    /**
+     * Dropbox API v2 requires a POST request to simply get user info
```

Comment:
Phpdocs summary should end in either a full stop, exclamation mark, or question mark.